package.json

```diff
@@ -125,7 +125,7 @@
     "lint:changelog": "node ./scripts/lint-changelog.js",
     "lint:prettier": "prettier \"**/*.{md,json,yml,html,css}\" --check",
     "lint:dist": "eslint --no-eslintrc --no-ignore --env=es6,browser --parser-options=ecmaVersion:2016 \"dist/!(bin-prettier|index|third-party).js\"",
-    "lint:spellcheck": "cspell {bin,scripts,src,website}/**/*.js {docs,website/blog,changelog_unreleased}/**/*.md",
+    "lint:spellcheck": "cspell **/* .github/**/*",
```

Comment:
I guess we should put quotes around `**/*`, otherwise it's expanded by the shell.

Reply:
Right

Reply:
`.github/workflows/*` must have been excluded for this reason.

Reply:
I guess because `node-glob` ignore `dot` file by default https://github.com/isaacs/node-glob#options

Can't see any override https://github.com/streetsidesoftware/cspell/blob/master/packages/cspell-glob/src/GlobMatcher.ts